### PR TITLE
Fontawesomeupgrade

### PR DESCRIPTION
**Pull request by AndrewIndeche**

### CODE REVIEW

File: common/components/constants/LinkConstants.js [hunk 1]
```diff
@@ -52,6 +57,11 @@ export const LinkDisplayConfigurationByUrl: $ReadOnlyArray<LinkSourceDisplayConf
     sourceDisplayName: "LinkedIn",
     iconClass: GlyphStyles.LinkedIn,
   },
+  {
+    sourceUrlPattern: new RegExp("Figma.com", "i"),
+    sourceDisplayName: "Figma",
+    iconClass: GlyphStyles.Figma,
+  },
```
Comment (marlonkeating) [inline common/components/constants/LinkConstants.js:64]: The only difference between this and the change at line 30 is capitalization, which should be covered by the "i" flag.

File: common/components/constants/LinkConstants.js [hunk 2]
```diff
@@ -63,6 +73,7 @@ export const LinkTypes: Dictionary<string> = {
   DESIGN: "link_design",
   TWITTER: "social_twitter",
   FACEBOOK: "social_facebook",
+  Figma: "link_facebook",
```
Comment (marlonkeating) [inline common/components/constants/LinkConstants.js:76]: You don't need to add an entry here.  It's covered under the DESIGN type.

File: common/components/constants/LinkConstants.js [hunk 3]
```diff
@@ -123,6 +134,13 @@ export const DefaultLinkDisplayConfigurations: KeyValuePair<LinkSourceDisplayCon
         iconClass: GlyphStyles.FacebookSquare,
       },
     ],
+    [
+      LinkTypes.FIGMA,
+      {
+        sourceTypeDisplayName: "Figma",
+        iconClass: GlyphStyles.Figma,
+      },
+    ],
```
Comment (marlonkeating) [inline common/components/constants/LinkConstants.js:143]: Again, don't need this, it lives under DESIGN

File: common/components/utils/glyphs.js [hunk 4]
```diff
@@ -46,6 +48,7 @@ export const GlyphStyles: { [key: string]: string } = {
   Palette: "fas fa-palette", // https://fontawesome.com/v5.15/icons/palette?style=solid
   HeartEmpty: "far fa-heart", // https://fontawesome.com/v6.0/icons/heart?s=regular
   HeartFilled: "fas fa-heart", // https://fontawesome.com/v6.0/icons/heart?s=solid
+  Figma: "fab fa-figma", // https://fontawesome.com/icons/figma?s=brands
```
Comment (marlonkeating) [inline common/components/utils/glyphs.js:51]: Duplicate of line 8

File: docker-compose.yml [hunk 5]
```diff
@@ -6,7 +6,7 @@ services:
     env_file:
       - ./.env
     ports:
-      - "5432:5432"
+      - "5438:5438"
```
Comment (marlonkeating) [inline docker-compose.yml:9]: Why did you need to change this?

**Review by marlonkeating — CHANGES_REQUESTED** [review]

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Currently the package.json imports aren't being used, we're still using the old import in 
https://github.com/DemocracyLab/CivicTechExchange/blob/fc545a165088113ac6d97ceb4da4ddaa00dd610c/civictechprojects/templates/new_index.html#L21

If you try putting a figma link in a project, you'll see that no icon shows up.

**Review by marlonkeating — CHANGES_REQUESTED** [review]
It seems to work fine, but in the change we've got v5.12.1 linked in new_index.html, and v.6.1.1 linked to in package.json.  If we really do need both, their versions should match up.

File: common/components/chrome/SocialFooter.jsx [hunk 6]
```diff
@@ -39,7 +39,7 @@ class SocialFooter extends React.Component {
             <i className={Glyph(GlyphStyles.LinkedIn, GlyphSizes.X3)}></i>
           </a>
         </div>
-      </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (AndrewIndeche) [reply common/components/chrome/SocialFooter.jsx]: The spacing is only reflecting when versioning,i can't see and spacing locally

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Looks good.  Go ahead and remove the yarn.lock changes now that package.json is no longer being changed.

**Comment by AndrewIndeche** [comment]
> Looks good. Go ahead and remove the yarn.lock changes now that package.json is no longer being changed.

completed

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**